.github/workflows/build-redpanda.yml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2023 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: build-redpanda
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: build redpanda
+    runs-on: ubuntu-latest-64
+    strategy:
+      matrix:
+        os: ["fedora:38", "ubuntu:mantic"]
```

Comment:
I tried fc39, but it looks like its going to take some effort. Since most of the team uses fedora, I suspect it will get fixed in short order, but its probably too big of a project to tackle in this PR.